jenkins/deployment-strategy.md

```diff
@@ -0,0 +1,157 @@
+# 🚀 2060.io Deployment Strategy – Detailed Guide
+
+This document describes the detailed procedure to deploy core components of the **2060.io** infrastructure. It covers both automated deployments using **Jenkins with Blue-Green strategy** and manual deployments via **Helm**. The goal is to standardize and clarify every step to ensure safe and reliable deployments.
+
+---
+
+## 📦 Pre-requisites
+
+Before starting any deployment, ensure the following:
+
+### 🔐 Access Requirements
+
+- Access to the GitHub repositories: [2060-core-deploy-dev](https://github.com/2060-io/2060-core-deploy-dev), [2060-core-deploy-prod](https://github.com/2060-io/2060-core-deploy-prod), and [2060-demos-deploy-dev](https://github.com/2060-io/2060-demos-deploy-dev)
+- Access to [**Jenkins**](https://jenkins.dev.2060.io) with permissions to execute deployment jobs
+- Access to **Kubernetes cluster**
+- DockerHub read/push credentials (as needed)
+
+### 💻 Tools Installed
+
+- `kubectl`
+- `helm` v3+
+- `git`
+- Valid `KUBECONFIG` file
+- Jenkins UI access
+
+---
+
+## 🔄 Blue-Green Deployment via Jenkins
+
+This strategy allows you to deploy a new version in parallel ("blue" or "green") and switch traffic only when the deployment is verified.
+
+### 📘 Naming Convention
+
+All Helm releases **must be prefixed with `2060-`**, followed by the service name (which must match the folder name under `deployments/`) and the color suffix (either `-blue` or `-green`).
```

Comment:
I'm curious about why Helm releases must be prefixed by 2060? It is just a legacy thing (previously we were sharing Kubernetes with other organizations)?